task/github-set-status/0.2/README.md

```diff
@@ -77,3 +79,45 @@ spec:
     - name: TARGET_URL
       value: https://tekton/dashboard/taskrun/log
 ```
+
+
+## Usage for Basic authentication
+
+Make sure the token is fabricated by base64 encoding the username and password with a semicolon in between.
+Example:
+```
+Username: githubuser
+Password: reallyinsecure
+Pseudo-code: base64(username+":"+password)
+Result: Z2l0aHVidXNlcjpyZWFsbHlpbnNlY3VyZQ==
+```
```

Comment:
Perhaps a small shell script would be better than pseudo-code? 
Quotes around the colon could break things depending on the language used...

Reply:
Added a small script.